docs/howto/serialize_poro.md

```diff
@@ -21,12 +24,56 @@ class MyModel
 end
 ```
 
-Fortunately, ActiveModelSerializers provides a [`ActiveModelSerializers::Model`](https://github.com/rails-api/active_model_serializers/blob/master/lib/active_model_serializers/model.rb) which you can use in production code that will make your PORO a lot cleaner.  The above code now becomes:
+### Inheriting ActiveModelSerializers::Model
+Fortunately, ActiveModelSerializers provides a [`ActiveModelSerializers::Model`](https://github.com/rails-api/active_model_serializers/blob/master/lib/active_model_serializers/model.rb) which you can use in production code that will make your PORO a lot cleaner.
+The above code now becomes:
 ```ruby
 # my_model.rb
 class MyModel < ActiveModelSerializers::Model
   attr_accessor :id, :name, :level
 end
 ```
 
-The default serializer would be `MyModelSerializer`.
\ No newline at end of file
+The default serializer would be `MyModelSerializer`.
+
+### Serializing an instance
+Sometimes you don't want to create yet another class just to serialize an object.
+Well, athough it is recommended to have defined classes for what you serialize,
+this is possible as well:
+
+Given the following serializer:
+
+```ruby
+class Api::V1::UserSerializer
+  type :user #this is necessery, otherwise AMS will show your instance class name
+
+  attributes :id, :name
+```
+
+```ruby
+class Api::V1::UsersController
+  def show
+    #this is just an example
+    an_instance = OpenStruct.new(id: 1, name: 'Just an example')
+
+    an_instance.singleton_class.send(:alias_method, :read_attribute_for_serialization, :send)
+
+    render jsonapi: an_instance, serializer: Api::V1::UserSerializer
+  end
+end
+```
+
+If your object is just a hash you can do the following trick:
```

Comment:
I would like to strongly recommend not trying to serialize primitives like Hashes.  AMS is intended to models.  There's a lot of parts of the AMS contract that would be weird to have on a hash.  https://github.com/rails-api/active_model_serializers/blob/master/lib/active_model/serializer/lint.rb